docs/components/BlockieAvatar.md

```diff
@@ -0,0 +1,35 @@
+---
+sidebar_position: 6
+---
+
+# BlockieAvatar
+
+Show a blockie (bar code profile icon) component for a public address.
+
+If the address has an ENS avatar linked to it, will show it. Otherwise, will show the generated blockie.
+
```

Comment:
This might be misleading 😅...Since it will show the ENS avatar only if you pass in `ensImage` and if you pass in any random image link it will show that image

--- 


Also the component this component is kind of "dumb" component like it does not have much logic init just shows what you pass in, it was mainly added for our internal prupose of cusomt RainbowKitButton and passing it to `RainbowtkitProviders`. 

I was wondering how to better explain this component may be : 

# BlockieAvatar

Show a blockie (bar code profile icon) component for a public address.


## Usage

```ts
<BlockieAvatar
  address={address}
  size={24}
/>
```

## Props

- `address` => ...

- `ensImage` (optional) => ENS avatar image link to show instead of blockie bar code ? .

- `size` => Width and Height in pixels (square).

Reply:
Was looking at these components:

![image](https://github.com/scaffold-eth/se2-docs/assets/2486142/d1fafa68-dca9-4fa9-ad48-cd4e8c8aaafd)


It's a bit weird that these are at the **same level** as the Balance / Address / Inputs. I feel they are a bit of "internal" components. Well, maybe the `RainbowKitCustomConnectButton` was worth explaining.... but not sure about the other ones.

Just thinking out loud, maybe it doesn't hurt to have them there (in the same level).

What do you all think?

Reply:
Oh, now I see Shiv's comments too :D

Reply:
Thanks for the suggestion! 🔥 

I think you are right, if the components are internal and devs won't reuse them to build their apps, is better to focus on main ones, avoiding "noise" in the Docs.

I've just deleted BlockieAvatar, Faucet, FaucetButton, Bytes32Input and BytesInput, and moved all the remaining ones to "Components" folder, don't think is worth it to separate Input Components for now, since there will be only 6-7 components in Docs.